nautobot/ipam/tables.py

```diff
@@ -508,6 +509,7 @@ class Meta(IPAddressTable.Meta):
         default_columns = (
             "pk",
             "address",
+            "parent__namespace",
```

Comment:
Is this the correct table class to add this to? Or should we instead (or also) be adding `parent__namespace` to the `default_columns` of `IPAddressTable` class?

Reply:
Yup, This is the correct one, also `IPAddressTable` currently do not have `default_columns`